### PR TITLE
DDF-1525 

### PR DESCRIPTION
**Pull request by gordocanchola**
This is the PR for #313 with the correct branch name.

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/codice/ddf/pull/356" target="_blank">ReviewNinja

**Comment by stustison** [comment]
Can you amend your commit to have the correct DDF ticket number?

**Comment by stustison** [comment]
:+1:

**Comment by djblue** [comment]
:+1:

File: platform/admin/docs/src/main/resources/ManContents.adoc [hunk 1]
```diff
@@ -124,46 +141,46 @@ Do NOT deselect/uninstall the Platform App or the Admin App. Doing so will disab
...
+Platform App, Admin App, and Security Services App CANNOT be selected or unselected as it is installed by default and can cause errors if removed.
+**Security Services App appears to be unselected upon first view of the tree structure, but it is in fact automatically installed with a later part of the installation process.
+====
 
 General Configuration Page
 
 image::general_configuration.png[General Configuration,${image.width}]
 
-Installation Profile Page
+General Configuration Page (Certs)
```
Comment (pklinef) [inline platform/admin/docs/src/main/resources/ManContents.adoc]: Replace Certs with Certificates everywhere in documentation.

File: platform/admin/ui/src/main/webapp/js/models/Installer.js [hunk 2]
```diff
@@ -139,7 +140,22 @@ define([
...
+                        $.ajax({
+                            type: 'GET',
+                            url: that.rebootUrl,
+                            dataType: 'JSON'
+                        }).done(function () {
+                            window.setTimeout(function () {
+                              window.location.href = that.get("redirectUrl");
+                            }, 30000);
+                        }).fail(function () {
+                            alert("Error trying to restart system. Please manually restart the system.");
```
Comment (pklinef) [inline platform/admin/ui/src/main/webapp/js/models/Installer.js]: So far we have not used `alert` in our web apps.  Usually we will display the error on the page with the error styling.  This `alert` should be replaced and instead look like the configuration model and insecure defaults errors.

File: platform/admin/ui/src/main/webapp/js/models/installer/CertsModel.js [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+define([
+    'marionette',
+    'underscore',
+    'backbone',
+    'jquery'
+], function (Marionette, _, Backbone, $) {
+
+    var CertsModel = Backbone.Model.extend({
+        defaults: {
+            hostName: undefined,
```
Comment (pklinef) [inline platform/admin/ui/src/main/webapp/js/models/installer/CertsModel.js]: We have not done this in the past but I like this.  Makes is easy to see what fields we expect to be in the model.

File: platform/admin/docs/src/main/resources/ManContents.adoc (hunk 1 above)
Comment (gordocanchola) [inline platform/admin/docs/src/main/resources/ManContents.adoc]: Replaced Certs with Certificates

File: platform/admin/ui/src/main/webapp/js/models/Installer.js (hunk 2 above)
Comment (gordocanchola) [inline platform/admin/ui/src/main/webapp/js/models/Installer.js]: Alerts are in another place in the installer. Created a ticket to remove the alerts.

**Pull request merged**